### PR TITLE
Fix Rsync when used locally

### PR DESCRIPTION
**Pull request by mablae**
This is a bugfix of #53

### CODE REVIEW

**Comment by oanhnn** [comment]
I think this is ok!
@deployphp/maintainer

**Comment by elfet** [comment]
@johnny-bit ping

**Comment by johnny-bit** [comment]
IMO - is OK :+1:

File: recipes/rsync.php [hunk 1]
```diff
@@ -125,11 +125,18 @@
...
+    }
+
+    $server = $server->getConfiguration();
     $host = $server->getHost();
     $port = $server->getPort() ? ' -p' . $server->getPort() : '';
     $identityFile = $server->getPrivateKey() ? ' -i ' . $server->getPrivateKey() : '';
     $user = !$server->getUser() ? '' : $server->getUser() . '@';
 
     runLocally("rsync -{$config['flags']} -e 'ssh$port$identityFile' {{rsync_options}}{{rsync_excludes}}{{rsync_includes}}{{rsync_filter}} '$src/' '$user$host:$dst/'", $config['timeout']);
+
```
Comment (oanhnn) [inline recipes/rsync.php]: Can you remove a blank line before function end?

Comment (mablae) [inline recipes/rsync.php]: Sure

**Comment by kenjis** [comment]
I think it is okay. Would you please merge it?

**Comment by ytetsuro** [comment]
If the this PR to merge.
everyone will be happy !!

**Comment by elfet** [comment]
ping @johnny-bit

**Comment by johnny-bit** [comment]
@elfet I already said it's OK :wink:

**Comment by johnny-bit** [comment]
hey @elfet - while you're at ti can you tag release? 3.1.0 has rsync unusable, because only in #48 paths were fixed and in order to use rsync properly one has to use dev-master version :) taging release would immediatelly fix #52 and #65 :)

**Comment by elfet** [comment]
Ok!

**Pull request merged**